.travis.yml

```diff
@@ -1,5 +1,7 @@
 dist: trusty
 language: python
+isntall:
```

Comment:
Typo?

Reply:
I remove back this build phase (using the default).